tests/integration/queries/test_model_group.py

```diff
@@ -206,3 +209,19 @@ def check_annotation_metrics(result):
     assert isinstance(result.model_level_metrics[0].macro_f1, float)
     assert isinstance(result.model_level_metrics[0].weighted_f1, float)
     assert result.retrain_for_metrics is False
+
+
+def test_update_model_group_settings(indico, org_annotate_model_group):
+    client = IndicoClient()
+    model_training_options = {"use_autolabeled_data": True, "use_partial_data": True}
```

Comment:
we should test with the `ModelOptions` object

Reply:
ModelOptions is just the return type model_training_options/predict_options are dict inputs

We test that the return type is ModelOptions and test that certain parameters are populated is that enough or should we cover other cases just to be safe?

Reply:
I'm just easily confused haha. 

But, if it's possible, could we provide typed input for users?

Reply:
We could but model_training_options and predict_options take in different inputs depending on what model type they're used on. We would probably be creating like 4 different types and adding logic to determine which type to accept. The documentation is updated to inform the user of which settings to use so maybe that's enough instead?

Reply:
yeah, fair, maybe something to take for a future refactoring and we can add a readme guide for now?

Reply:
ah yep ill make a ticket for that